lazyloader.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * This is 100% not cross-browser.
+ * We like to use cutting edge technologies in the organization.
+ */
+export const load = source => {
+  const img = new Image();
+  img.src = source;
+
+  return img.decode().then(() => img);
```

Comment:
Ref: https://html.spec.whatwg.org/multipage/embedded-content.html#dom-img-decode

This is the best way to handle loading images asynchronously as the decoding doesn't happen in the main thread but instead a worker is used for it.

Reply:
Thoughts on a `nomodule` fallback?

Reply:
Nit: this could benefit from the cache, state, view, dispatch pattern made popular in 2011.

Reply:
@kirillzakharov You 100% right. We need a fallback to ancient browsers that aren't capable to keep up with the spec. I will provide a fix for it.

Reply:
Makes sense. Or we can use react, redux, recompose, reselect, jest and mongodb.